### ISSUE TITLE
anti adblock killer doesnt work on this page

### ISSUE THREAD

**Issue by acetriper** [reek/anti-adblock-killer#2744]
http://www.latimes.com/science/sciencenow/la-sci-sn-science-skepticism-psychology-20170120-story.html

**Comment by uBlock-user** [comment]
cannot reproduce

**Comment by mapx-** [comment]
see if helps (add this filter)
`||tribdss.com^$domain=latimes.com`

**Comment by acetriper** [comment]
The filter didnt work, still getting this:
![screen shot 2017-01-24 at 13 50 25](https://cloud.githubusercontent.com/assets/24705126/22247859/3c425336-e23c-11e6-8cc8-6f9c70379d42.png)

**Comment by uBlock-user** [comment]
add - `||$third-party,domain=laimes.com`

**Comment by uBlock-user** [comment]
add - `||$third-party,domain=latimes.com`

**Comment by acetriper** [comment]
thanks! does the job :)
(I hide an extra element on bottom right that couldnt be "x'ed"

**Comment by jspenguin2017** [comment]
@uBlock-user That seems to break videos and images.

**Comment by jspenguin2017** [comment]
My solution: 
```JavaScript
if (a.domCmp(["latimes.com"])) {
    //Watch and remove block screen
    a.filter("addEventListener", /return\"undefined\"\!\=\=typeof\ e/);
    a.css("html,body {overflow:scroll;}");
    a.observe("insert", function (node) {
        if (node.innerHTML &&
            node.innerHTML.includes("please turn off your ad blocker or subscribe")) {
            node.remove();
        }
    });
}
```

**Comment by uBlock-user** [comment]
Yes, whatever breaks, user has to create an exception for that in uBO.

**Comment by jspenguin2017** [comment]
@uBlock-user I think the anti-adblock script and the main script are together... So you can't just block that or images, videos, and comments won't work.

**Comment by okiehsch** [comment]
`||tribdss.com/meter/assets$script,domain=www.latimes.com` works on my end and 
images, videos and comments still work

**Comment by jspenguin2017** [comment]
@okiehsch Hum, you are right. Thanks for your solution.

**Comment by mapx-** [comment]
already above:  https://github.com/reek/anti-adblock-killer/issues/2744#issuecomment-274778734

**Comment by jspenguin2017** [comment]
@mapx- I think your rule blocks images and videos. Anyway, whatever works.

**Comment by okiehsch** [comment]
@mapx- images and videos still work with `||tribdss.com^$domain=latimes.com`, but
comments are broken on my end.

**Comment by uBlock-user** [comment]
`latimes.com` appears to have already bee patched by Easylist, so there's no need.

**Comment by 22350** [comment]
none of these are working any more.  easylist either

**Comment by jspenguin2017** [comment]
@22350 It still works for me.

**Comment by 22350** [comment]
If I drop the following link into AdBlock chrome and it works:

||tribdss.com/meter/assets$script,domain=www.latimes.com

**Comment by jspenguin2017** [comment]
@22350 Ah... I thought you meant that rule doesn't work.
